packages/website/docs/Checkbox.md

```diff
@@ -1,29 +1,57 @@
-## Guidelines
+# Best Practices
 
-### General
+Use checkboxes to allow user to select multiple options or none of them.
 
--   Whereas checkboxes can be used to toggle options on and off, using a [toggle switch](#/components/FlatSelect) or [radio buttons](#/components/Radio) is preferable.
--   A checkbox set should always have a title.
--   Always put the most common options at the top of the checkbox list.
--   If more than 7 options (checkboxes) are needed, it is preferable to group checkboxes logically under more than one set of options such that the set label represents the grouping logic.
+Use a stand-alone checkbox to toggle a single option on or off and when the effect is not immediate.
 
-### Label
+-   E.g. if a final confirmation is required to apply the effect.
```

Comment:
For instance, you could use a checkbox to get final confirmation from a user before changes are applied.

Reply:
Use a stand-alone checkbox to toggle a single option on/off if the result is not immediately visible, e.g., when a final confirmation from the user is required to apply changes.

Conversely, if the result is applied immediately, such as a change of display, use a [Toggle](#/form/FlatSelect) instead.